webcomponents/select/omnia-select.js

```diff
@@ -64,6 +65,8 @@ class OmniaSelect extends HTMLElement {
 
         const newValueIndex = this._options.map(obj => obj.value).indexOf(this._value) + 1;
 		this._select.selectedIndex = newValueIndex >= 0 ? newValueIndex : 0;
+	    if (this._options[newValueIndex - 1])
```

Comment:
Why store the text in a property?
The evaluation could be made in the getter. This way, it only be calculated when requested.